libs/common/src/constants/facility-type.ts

```diff
@@ -1,8 +1,6 @@
-const FACILITY_TYPE = {
+export const FACILITY_TYPE = {
```

Comment:
just a comment to say how fancy is github realising this file has moved and displaying it nicely

Reply:
I thought the same!